### PR TITLE
Coach classes string stubbing

### PR DESCRIPTION
**Pull request by DXCanas**
### Summary

Stubbing strings according to latest mocks, going off of a string change/add audit I outlined here (red is visual, blue means probable a11y implications, green is css-only string additions):
string audit class coaches update1

### Reviewer guidance

please ignore the kill comments. In place to remind me of areas for cleanup for upcoming coach-classes development.

Created more strings instead of modifying existing ones. If they're going to have to be retranslated anyway, I'd rather provide a little more context in the string's label. 

@radinamatic are the types of labels I'm going for useful?

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue [hunk 1]
```diff
@@ -103,6 +103,8 @@
       learners: 'Learner reports',
       name: 'Name',
       group: 'Group',
+      channelIconColumnHeader: 'Channel icon',
+      userIconColumnHeader: 'User icon',
```
Comment (DXCanas) [inline kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue:107]: @radinamatic in situations like this:
https://github.com/DXCanas/kolibri/blob/33edc0261de6d7e5b94b5fea1a5dc942afadbac9/kolibri/plugins/coach/assets/src/views/class-list-page/index.vue#L13

Where there is a column entirely dedicating to displaying icons, is it helpful/appropriate to add an invisible column header to label that column, like so?

Comment (radinamatic) [reply kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue:107]: Kudos for trying to thoroughly think this through from the user perspective @david! :+1: 
I cannot run this locally to test in place (hopefully my new testing machine will arrive tomorrow, and I can be back on track), but from the mockups I infer that the icons in the column are all the same, right? All users, or classes, etc.. In that case I would say it's safe to assume that the icons have mostly decorative function and we can make them invisible, which renders the invisible headers for those columns unnecessary :slightly_smiling_face:

File: kolibri/plugins/facility_management/assets/src/views/class-learner-enroll-page.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'classLearnerEnrollPage',
```
Comment (jonboiser) [inline kolibri/plugins/facility_management/assets/src/views/class-learner-enroll-page.vue:13]: "class learner enroll" page sounds like a bag of words that humans wouldn't say.

On the mocks, the header of this page is "Enroll learners into 'Class'". Could we name it something close to that like "EnrollLearnersPage" (named after action/verb) or "LearnerEnrollmentPage" (named after workflow).

Comment (DXCanas) [reply kolibri/plugins/facility_management/assets/src/views/class-learner-enroll-page.vue:13]: I agree, I was sort of shoehorning the word "class" in to denote that it's a class-related page within the `facility_management` plugin

File: kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue (hunk 1 above)
Comment (DXCanas) [reply kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue:107]: I don't think making the icons themselves invisible is an option on the table :/ Did you mean invisible to screenreaders?

Comment (radinamatic) [reply kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue:107]: Yes, sorry for being imprecise, that's exactly what I meant!

Comment (DXCanas) [reply kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue:107]: You know the best way to go about making them invisible to screen readers? I'm unclear on how they go through tables

**Comment by DXCanas** [comment]
@jonboiser recommended closing this PR and just extracting strings from his branch [here](https://github.com/jonboiser/kolibri/tree/class-coaches-1).

**Pull request closed without merging**